index.js

```diff
@@ -22,6 +22,23 @@ const removeElements = elements => {
 	}
 };
 
+const disableAnimations = options => {
+	const defaults = {transform: false, animation: true, transition: true};
+	options = {...defaults, ...options};
+
+	const transform = options.transform ? 'transform: initial !important;' : '';
+	const animation = options.animation ? 'animation: initial !important;' : '';
+	const transition = options.transition ? 'transition: initial !important;' : '';
+
+	const rule = `*,:after,:before{${transform}${animation}${transition}}`;
+
+	const style = document.createElement('style'); // eslint-disable-line no-undef
+	document.body.append(style); // eslint-disable-line no-undef
+
+	const {sheet} = style;
+	sheet.insertRule(rule);
```

Comment:
```suggestion
	style.sheet.insertRule(rule);
```

You don't need `sheet` for anything else, so no point in a variable for it.